subsys/net/l2/ethernet/ethernet.c

```diff
@@ -136,9 +137,18 @@ static void ethernet_update_rx_stats(struct net_if *iface,
 #endif /* CONFIG_NET_STATISTICS_ETHERNET */
 }
 
+static inline bool is_vlan_tag_stripping_enabled(
```

Comment:
just give the net_if iface as a parameter directly.

also rename that function eth_is_vlan_tag_stripped(), it's nicer imo.